filereader/texttemplate/texttemplate.go

```diff
@@ -21,9 +22,27 @@ func Parse(filename string, funcs template.FuncMap) (*template.Template, error)
 			}
 			return content, nil
 		},
+		"toJSON": func(v interface{}) (string, error) {
```

Comment:
@redbaron I believe we can expect it to remain to be existing because it is used widely in many helm charts.